src/views/ManageList.jsx

```diff
@@ -1,10 +1,10 @@
 import { AddItems } from '../components/AddItems';
 import { ShareList } from '../components/ShareList';
 
-export function ManageList() {
+export function ManageList({ data }) {
 	return (
 		<div>
-			<AddItems />
+			<AddItems data={data} />
```

Comment:
I would suggest renaming the `data` prop to something more descriptive like `items`. `Data` is really generic and it's hard to infer what data this prop actually holds